app/views/users/index.html.erb

```diff
@@ -1,3 +1,4 @@
+<h1>User profile</h1>
 <% @users.each do |user|%>
   <div class='container'>
   <div class="row mb-3 shadow-lg p-3 mb-5 bg-body-tertiary rounded d-flex" style="max-width: 540px;">
```

Comment:
- [OPTIONAL] I would like to advise you to use partials to render this page. Partials are used to render similar UI features that can be found across different pages. Kindly render it. 👍 😎 


#### Hint: You can read this part of the [documentation](https://guides.rubyonrails.org/layouts_and_rendering.html#:~:text=3.4%20Using%20Partials,its%20own%20file.) 👍 😎